articles/install-guide/vs-2017.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Installing the Q# development environment for VS 2017
+author: chgranad
+uid: microsoft.quantum.install.vs-2017
+ms.author: christopher.granade@microsoft.com
+ms.date: 7/31/2018
+ms.topic: article
+---
+
+# Installing the Quantum Development Kit for Visual Studio 2017 #
+
+## Installing Visual Studio 2017 (Windows 10 only)
+
+> [!NOTE]
+> The Microsoft Quantum Development Kit does not support Visual Studio for Mac.
+> To install on macOS, please follow the instructions for [using Visual Studio Code](xref:microsoft.quantum.install.vs-code).
+
+If you do not have Visual Studio installed, you can download Visual Studio 2017 Community Edition for free.
```

Comment:
Instead of providing instructions why don't we just point people to this URL:
https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio

They are better documented and easier to follow, and owned by VS.

If we do, there are two important things need to highlight:
1. People need to have VS 2014 Update 4 or newer (older version will not work with .net core)
2. People need to select the **.NET Core cross-platform development** workload.

Reply:
If we point to the URL, installation for Windows 10 consists only on installing the VSIX.